src/chapters/interfaces-and-polymorphism/interfaces-and-abstract-classes.rst

```diff
@@ -0,0 +1,41 @@
+Interfaces and Abstract Classes
+===============================
+
+We mentioned previously that interfaces share some characteristics with
+abstract classes. Recall that an abstract class is one declared with the
+``abstract`` keyword. You may not create an object from an abstract
+class and, like an interface, an abstract class is allowed to contain
+methods that only have signatures (that is, methods that don’t have
+implementation code).
+
+The main differences between interfaces and abstract classes are:
+
+#. You *implement* an interface, while you *extend* an abstract class. The net effect of this is 
+   that a class may implement interfaces while also extending a class. Note that while you can 
+   implement *more than one* interface, you can only extend *one* class.
+#. Abstract classes may contain non-constant fields, while interfaces can only contain constant 
+   fields.
+#. Abstract classes should be used to collect and specify behavior by related classes, while an 
+   interface should be used to specify related behaviors that may be common across unrelated 
+   classes.
```

Comment:
Can you elaborate on this point or maybe rephrase it? I think this final point is the one that is really driving home why we choose to use interfaces vs. abstract classes and it might be easy to miss the "related" vs. "unrelated". Maybe emphasizing those two words would be a simple fix that would help.